src/core/lib/backoff/backoff.h

```diff
@@ -21,63 +21,85 @@
 
 #include "src/core/lib/iomgr/exec_ctx.h"
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
-typedef struct {
-  /// const:  how long to wait after the first failure before retrying
-  grpc_millis initial_backoff;
-
-  /// const: factor with which to multiply backoff after a failed retry
-  double multiplier;
-
-  /// const: amount to randomize backoffs
-  double jitter;
-
-  /// const: minimum time between retries
-  grpc_millis min_connect_timeout;
-
-  /// const: maximum time between retries
-  grpc_millis max_backoff;
-
+namespace grpc_core {
+
+class Backoff {
```

Comment:
some documentation maybe about the class in general

Reply:
Done.